osd/default_scc_modification.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: Revert modifications to default Security Context Contraints",
+    "description": "Your cluster requires you to take action because there has been a change to the cluster's default Security Context Constraints (SCC). Without action, your cluster's SLA may be impacted, along with your ability to upgrade. Please do not modify default SCC, instead create new one as described here: https://docs.openshift.com/container-platform/4.8/authentication/managing-security-context-constraints.html. Please revert changes or contact support for more assistance.",
```

Comment:
Suggest: 
"The cluster's default Security Context Constraints (SCC) have been modified. Please revert the changes to the default SCC or contact support for more assistance. Your cluster's SLA and ability to upgrade may be impacted if you do not take action. Instead of modifying default SCC, it is recommended to create new SCC as described here: https://docs.openshift.com/container-platform/4.8/authentication/managing-security-context-constraints.html.",